src/js/components/user/user.js

```diff
@@ -45,8 +45,8 @@ class User extends MaharaBaseComponent {
             icon = defaultIcon;
             displayName = "";
         }
-
-
+        // add timestamp to query to prevent image caching, because we're using same name 
+        icon += `?${Date.now() || ''}`;
```

Comment:
It's user specific info. I'm afraid the most reliable thing would be the timestamp